nemo/collections/common/callbacks/nemomodelcheckpoint.py

```diff
@@ -0,0 +1,215 @@
+import os
```

Comment:
could you rename this file to nemo_model_checkpoint.py?

Reply:
Also, I don't think this should go under collections? maybe utils/callbacks ?

Reply:
Also, make sure to add our header whenever adding new files. You can copy the header from any other file in NeMo, just update the year to the current one.